cla-backend-go/signatures/service.go

```diff
@@ -333,3 +334,18 @@ func (s service) DeleteGithubOrganizationFromWhitelist(signatureID string, white
 
 	return gitHubWhiteList, nil
 }
+
+// Disassociate project signatures
+func (s service) RemoveProject(projectID string) error {
```

Comment:
Not deleting, but invalidating signature records - recommend we rename the function name.